neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/BaseNeo4jDialect.java

```diff
@@ -10,11 +10,17 @@
 import static org.hibernate.ogm.datastore.neo4j.query.parsing.cypherdsl.impl.CypherDSL.skip;
```

Comment:
In the commit message:
s/discriminatoir/discriminator/
s/Enitty/Entity/
s/is is/is/
s/remnove/remove/